static/app/components/modals/recoveryOptionsModal.spec.tsx

```diff
@@ -42,13 +42,14 @@ describe('RecoveryOptionsModal', function () {
 
   it('can redirect to recovery codes if user skips backup phone setup', async function () {
     renderComponent();
+    const skipButton = await screen.findByRole('button', {name: 'Skip this step'});
 
     expect(
       screen.queryByRole('button', {name: 'Get Recovery Codes'})
     ).not.toBeInTheDocument();
```

Comment:
logically this test flows a lot better with the findBy at the top 👍  since queryBy can't wait for things